policykit/redditintegration/views.py

```diff
@@ -87,10 +87,22 @@ def oauth(request):
     return response
 
 @csrf_exempt
-def initCommunity(request):
+def getStarterKit(request):
```

Comment:
also generally in python we don't use camelcase. init_community would be more "pythonic"